tools/perftests_ci/perftests_plot.py

```
@@ -0,0 +1,132 @@
+            
+    json_obj1 = json.loads(f1.read())
+    json_obj2 = json.loads(f2.read())
+
+    test_names = []
+    category = []
+    samples = []
+    
+    header_flag = True
+    
+    i = 0
+    for test1, test2 in zip(json_obj1["outputs"], json_obj2["outputs"]):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)